tsdb/db.go

```diff
@@ -639,9 +639,11 @@ func open(dir string, l log.Logger, r prometheus.Registerer, opts *Options, rngs
 	if err := MigrateWAL(l, walDir); err != nil {
 		return nil, errors.Wrap(err, "migrate WAL")
 	}
-	// Remove garbage, tmp blocks.
-	if err := removeBestEffortTmpDirs(l, dir); err != nil {
-		return nil, errors.Wrap(err, "remove tmp dirs")
+	for _, tmpDir := range []string{walDir, dir} {
+		// Remove garbage, tmp blocks.
```

Comment:
```suggestion
		// Remove tmp dirs.
```
Since its removing block + tmp in checkpoint and WAL, its good to update the comment.